### ISSUE TITLE
Add a "explain diff" button in the diff view that will use AI to describe the diff in natural language.

### ISSUE THREAD

**Issue by kentcdodds** [epicweb-dev/epicshop#142]
This would just be so cool.

**Comment by kentcdodds** [comment]
Of course, because this runs locally, people will have to provider their own OpenAI token, but we can explain how to do this.

**Comment by kettanaito** [comment]
Isn't this description the goal of the "completed" video/description? 

> Of course, because this runs locally, people will have to provider their own OpenAI token

I'd argue this is a high cost for a low gain.

**Comment by kettanaito** [comment]
What can have a bit bigger value is to explain the diff between the _user's solution_ and the official solution. Still not sure if it justifies the whole roundtrip of providing your own OpenAI token.

**Comment by kentcdodds** [comment]
This is now handled by the epicshop mcp server
